Source/Core/InputCommon/ControllerInterface/evdev/evdev.h

```diff
@@ -0,0 +1,84 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <libevdev/libevdev.h>
+
```

Comment:
string and vector should also be included here to get rid of indirect inclusion.